### ISSUE TITLE
implementation of GIN architecture

### ISSUE THREAD

**Issue by thegodone** [aimat-lab/gcnn_keras#15]
would be nice to make https://github.com/weihua916/powerful-gnns available in your Keras toolkit.

**Comment by PatReis** [comment]
Thanks, I will add it.

**Comment by thegodone** [comment]
is it ready for testing ?

**Comment by thegodone** [comment]
can you also make one on Lipop dataset too ?

**Comment by PatReis** [comment]
I have tried to make it identicial to the reference implementation with a BatchNormMLP, and it reaches the 76% accuracy, however, not always (even for constant split). I checked the reference code with same hyper-parameters and found the same behaviour. 
![gin_proteins](https://user-images.githubusercontent.com/64005383/124434309-041ebc00-dd74-11eb-88b2-df7e584e2163.png)
